main.py

```diff
@@ -69,7 +69,10 @@ def echo(message, history, state):
                     previous_summary=state['summary'], 
                     latest_conversation=str({"user": message['text'], "assistant": model_response})
                 )
-            ]
+            ],
+            config={'response_mime_type': 'application/json',
+                'response_schema': list[SummaryResponses],
```

Comment:
we don't want a list of summaries but a single summary.
hence we can get rid of `list` and just keep `SummaryResponse`